core/src/test/java/management/advertisement/AdvertisementHandlerTest.java

```diff
@@ -132,4 +135,30 @@ void getFilteredAdvertisementCampaigns_ShouldReturnCampaignsWithSpecificType() {
 		assertEquals(1, currentCampaigns.size());
 		assertEquals(AdvertisementType.RADIO_ADVERTISEMENT, currentCampaigns.get(0).advertisementData().type());
 	}
+
+	static Stream<Arguments> getCampaignFullCostArgs() {
```

Comment:
nie jestem pewnien czy używanie streamów do testów parametrycznych to jest dobre podejście ze wzgledów na optymalizacje no ale spoko

Reply:
mnie tak nauczono xd